pom.xml

```diff
@@ -46,6 +139,68 @@
 				<groupId>org.springframework.boot</groupId>
 				<artifactId>spring-boot-maven-plugin</artifactId>
 			</plugin>
+			<plugin>
+				<groupId>io.swagger</groupId>
+				<artifactId>swagger-codegen-maven-plugin</artifactId>
+				<version>2.3.1</version>
+				<executions>
+					<execution>
+						<goals>
+							<goal>generate</goal>
+						</goals>
+						<configuration>
+							<inputSpec>${project.basedir}/src/main/resources/swagger.yaml</inputSpec>
+							<language>spring</language>
+
+							<!-- pass any necessary config options -->
+							<configOptions>
+								<interfaceOnly>true</interfaceOnly>
+								<dateLibrary>joda</dateLibrary>
+							</configOptions>
+
+							<output>${project.build.directory}/generated-sources</output>
+							<apiPackage>com.johan.blignaut.entelect.vitalityincubator.api</apiPackage>
+							<modelPackage>com.johan.blignaut.entelect.vitalityincubator.model</modelPackage>
+							<invokerPackage>com.johan.blignaut.entelect.vitalityincubator</invokerPackage>
+						</configuration>
+					</execution>
+				</executions>
+			</plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-compiler-plugin</artifactId>
+				<version>3.8.0</version>
+				<configuration>
+					<source>11</source>
+					<target>11</target>
+					<forceJavacCompilerUse>true</forceJavacCompilerUse>
+				</configuration>
+			</plugin>
+			<plugin>
+				<!--
+                    To have the generated code included in the application build process we need to make the files
+                    visible to the maven compiler plugin. We use the build-helper-maven-plugin configured to use our generated sources.
+                -->
+				<groupId>org.codehaus.mojo</groupId>
+				<artifactId>build-helper-maven-plugin</artifactId>
+				<version>3.0.0</version>
+				<executions>
+					<execution>
+						<id>add-source</id>
+						<phase>generate-sources</phase>
+						<goals>
+							<goal>add-source</goal>
+						</goals>
+						<configuration>
+							<sources>
+								<source>
+									TODO
```

Comment:
Looks like the `configuation` tag isn't doing much, it could likely be removed.

Reply:
I'm not sure how it's working without a valid directory here😅. This should be the package where the generated code lives.

Reply:
I'm not sure either haha, but if I take it out, it breaks. Like I said, this was exactly copied from the code you gave me.